rust/src/filetracker.rs

```diff
@@ -106,6 +106,7 @@ impl FileTransferTracker {
 
     pub fn close(&mut self, files: &mut FileContainer, flags: u16) {
         if !self.file_is_truncated {
+            SCLogDebug!("closing file with id {}", self.track_id);
```

Comment:
Hmm I was assuming that if we compile for 'release' these would be no-ops. If that isn't the case I would like to make sure that gets done instead of removing debug statements.

Reply:
Does the formatting come after a check of the log level? We still have a function call overhead though. Would be nice to have a no-op I guess.

Reply:
unfortunately, no, it does cost both the format and the check for the loglevel. I will look into that, I think there is a simple solution for the loglevel `Debug`, using a feature

Reply:
Would be great if you can have a look at that, thanks Pierre!

Reply:
#3270